test/e2e_node/dual_stack_host_ips.go

```diff
@@ -137,5 +141,58 @@ var _ = common.SIGDescribe("Dual Stack Host IP [Feature:PodHostIPs]", func() {
 			err = podClient.Delete(context.TODO(), pod.Name, *metav1.NewDeleteOptions(30))
 			framework.ExpectNoError(err, "failed to delete pod")
 		})
+
+		ginkgo.It("should provide hostIPs as an env var", func() {
```

Comment:
do we need to split this commit ? or is just for make the review easier?

Reply:
I merged the two commits for e2e